src/Ledger/Ledger.lagda

```diff
@@ -60,9 +62,7 @@ private variable
 
 \begin{figure*}[h]
 \begin{code}[hide]
-open RwdAddr
-open DState
-open CertState
+open CertState; open DState; open LEnv; open LState; open RwdAddr; open TxBody
```

Comment:
Remove five instances of `open` for records from the figure at the expense of lots explicit use of accessors in the `LEDGER` transition rule below.  Is it worth it?  Is there a better way to avoid the `open` directives here?